Classes/Controller/NavigationController.php

```diff
@@ -135,8 +135,11 @@ public function mainAction(): ResponseInterface
                 $musicalStructure = $this->document->getCurrentDocument()->musicalStructure;
                 $musicalStructureInfo = $this->document->getCurrentDocument()->musicalStructureInfo;
                 for ($i = 1; $i <= $this->document->getCurrentDocument()->numMeasures; $i++) {
-                    if (array_key_exists($i, $musicalStructure) && array_key_exists($i, $musicalStructureInfo)) {
-                        $measureOptions[$i] = '[' . $i . ']' . ($musicalStructureInfo[$musicalStructure[$i]['measureid']]['orderlabel'] ? ' - ' . htmlspecialchars($musicalStructureInfo[$musicalStructureInfo[$i]]['orderlabel']) : '');
+                    if (isset($musicalStructure[$i])
+                        && array_key_exists($i, $musicalStructure)
+                        && isset($musicalStructureInfo)
+                        && array_key_exists($musicalStructure[$i]['measureid'], $musicalStructureInfo)) {
+                        $measureOptions[$i] = '[' . $i . ']' . ($musicalStructureInfo[$musicalStructure[$i]['measureid']]['orderlabel'] ? ' - ' . htmlspecialchars($musicalStructureInfo[$musicalStructure[$i]['measureid']]['orderlabel']) : '');
```

Comment:
This is quite redundant. If `$musicalStructure[$i]` is set we do not need to check if the array key `$i` exists, because it obviously does.
Instead, we need to check for array keys `measureid` and `orderlabel` before accessing them.

```suggestion
                    if (isset($musicalStructure[$i]['measureid'])
                        && isset($musicalStructureInfo[$musicalStructure[$i]['measureid']]['orderlabel'])) {
                        $measureOptions[$i] = '[' . $i . ']' . (!empty($musicalStructureInfo[$musicalStructure[$i]['measureid']]['orderlabel']) ? ' - ' . htmlspecialchars($musicalStructureInfo[$musicalStructure[$i]['measureid']]['orderlabel']) : '');
```

Reply:
Did this cause of PHPStan error https://github.com/kitodo/kitodo-presentation/actions/runs/16074314340 and thought that something had change with the configuration so i adjusted this.